src/app/zap-templates/app-templates.json

```diff
@@ -172,102 +172,6 @@
             "path": "../../../third_party/matter_sdk/src/app/zap-templates/templates/app/clusters-shared-Global-Ids.h.zapt",
             "name": "Shared global ids",
             "output": "clusters/shared/GlobalIds.h"
-        },
```

Comment:
Once ZAP changes are in, these templates will be restored with an added "static" key which will prevent their execution unless "static" is set the .zap file.